README.md

```diff
@@ -43,17 +43,22 @@ Full example
 cargo run -- --batch 109939 --network mainnet --l1-rpc https://rpc.ankr.com/eth
 ```
 
+If you need to update the verification key to the latest, run with the corresponding flag.
```

Comment:
btw - this means that once we update the verification keys in couple months, people will not be able to verify the older blocks? 

we should probably have some 'versioning' for the verification keys (mapping from protocol version -> verifictaion key)
(but this can be done in a separate PR)